config/config.go

```diff
@@ -156,18 +156,29 @@ func init() {
-	cfg.SetDefault("storage.memory.driver", "memory")                // defined for backward compatibility and to set defaults
-	cfg.SetDefault("storage.orientdb.driver", "orientdb")            // defined for backward compatibility and to set defaults
-	cfg.SetDefault("storage.orientdb.addr", "http://localhost:2480") // defined for backward compatibility and to set defaults
-	cfg.SetDefault("storage.orientdb.database", "Skydive")           // defined for backward compatibility and to set defaults
-	cfg.SetDefault("storage.orientdb.username", "root")              // defined for backward compatibility and to set defaults
-	cfg.SetDefault("storage.orientdb.password", "root")              // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.elasticsearch.driver", "elasticsearch")                                // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.elasticsearch.host", "127.0.0.1:9200")                                 // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.elasticsearch.bulk_maxdelay", 5)                                       // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.elasticsearch.index_age_limit", 0)                                     // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.elasticsearch.index_entries_limit", 0)                                 // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.elasticsearch.indices_to_keep", 0)                                     // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.memory.driver", "memory")                                              // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.orientdb.driver", "orientdb")                                          // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.orientdb.addr", "http://localhost:2480")                               // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.orientdb.database", "Skydive")                                         // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.orientdb.username", "root")                                            // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.orientdb.password", "root")                                            // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.objectstorage.driver", "objectstorage")                                // defined for backward compatibility and to set defaults
+	cfg.SetDefault("storage.objectstorage.endpoint", "http://s3.amazonaws.com")                    // defined for backward compatibility and to set defaults
```

Comment:
from this line you can remove the comment 

> // defined for backward compatibility and to set defaults

Reply:
i would suggest the backend is renamed to s3 rather than objectstorage (as there are several objectstorage standards such s3, swift)